examples/kmlFetchGood.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+See https://github.com/AnalyticalGraphicsInc/cesium-google-earth-examples/blob/master/LICENSE.md
+
+Fenway Model was used from the publicly available Model on Sketchup
+https://3dwarehouse.sketchup.com/model.html?id=21eee70f917ce34af7a0c817ab72c49b
```

Comment:
Did you check the [TOS](https://3dwarehouse.sketchup.com/tos.html)?  Are you sure we can use this?  In the past, we were not able to use models when this was Google's warehouse.